charms/slurmrestd/terraform/outputs.tf

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+output "app_name" {
+  value = juju_application.slurmrestd.name
+}
+
+output "provides" {
+  value = {
+    slurmctld = "slurmctld"
```

Comment:
Not sure if this should specify the integration name twice or if it should be the integration name and the interface name.

Reply:
It should be the integration name IIRC. It's effectively a map that just provides the names of the integrations that the charm provides. This output can then be consumed by a tf plan implementing the Slurm charm as a module.